routes/activities.js

```diff
@@ -55,7 +60,7 @@ router.put('/:id', (req, res) => {
 
 router.post('/', (req, res) => {
   const formData = req.body;
-  if (formData.title === null) {
+  if (formData.title === "") {
```

Comment:
you test if the fields are empty, but not if they are present. that was the purpose of the "== null": you need both tests ;)